plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -382,6 +382,116 @@ HAVING
 OPTION(RECOMPILE);
 `
 
+const sqlAzureDBPartQueryPeriod = `
+DECLARE @QueryData NVARCHAR(MAX) = ?1; --input parameter
+
+DECLARE @lastIntervalEndTimestamp BIGINT = 0;
+
+IF @QueryData != '' AND @QueryData IS NOT NULL
+	SET @lastIntervalEndTimestamp = CAST(@QueryData AS BIGINT);
+
+DECLARE @magicNumberDays BIGINT = 4294967296, @magicNumberSeconds BIGINT = 300, @secondsInDay INT = 86400;
```

Comment:
Why is Magic Number seconds 5 mins and not 15 mins? Any logic to it?

Reply:
Removed magic number. Now we return start_time and end_time as strings.